osmosis/assetlist.json

```diff
@@ -16314,6 +16314,84 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/ashLAB.png"
         }
       ]
+    },
+    {
+    "description": "Cosmos Airdrop Chat",
+    "extended_description": "Official Token of the Cosmos Airdrop community",
+    "denom_units": [
+        {
+            "denom": "factory/osmo1q77cw0mmlluxu0wr29fcdd0tdnh78gzhkvhe4n6ulal9qvrtu43qtd0nh8/cac",
+            "exponent": 0
+        },
+        {
+            "denom": "CAC",
+            "exponent": 6
+        }
+    ],
+    "type_asset": "sdk.coin",
+    "address": "osmo1q77cw0mmlluxu0wr29fcdd0tdnh78gzhkvhe4n6ulal9qvrtu43qtd0nh8",
+    "base": "factory/osmo1q77cw0mmlluxu0wr29fcdd0tdnh78gzhkvhe4n6ulal9qvrtu43qtd0nh8/cac",
+    "name": "Cosmos Airdrop Chat",
+    "display": "CAC",
+    "symbol": "CAC",
+    "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/CAC.svg"
+    },
+    "images": [
+        {
+            "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/CAC.svg"
+        }
+    ]
+    },
+        {
+      "description": "The native token of Nim Network.",
+      "extended_description": "Nim Network is a highly-adoptable AI Gaming chain that will provide the ultimate ecosystem for exploration and development of games at the intersection of Web3 and AI.",
+      "denom_units": [
+        {
+          "denom": "ibc/279D69A6EF8E37456C8D2DC7A7C1C50F7A566EC4758F6DE17472A9FDE36C4426",
+          "exponent": 0,
+          "aliases": [
+            "anim"
+          ]
+        },
+        {
+          "denom": "nim",
+          "exponent": 18
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/279D69A6EF8E37456C8D2DC7A7C1C50F7A566EC4758F6DE17472A9FDE36C4426",
+      "name": "NIM Network",
+      "display": "nim",
+      "symbol": "NIM",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "nim",
```

Comment:
nim is the wrong counterparty here. It should be the ibc dneom on dymensions--not anim on NIM.